vendor/omnipay/eway/src/Message/RapidDirectCreateCardRequest.php

```diff
@@ -12,10 +12,6 @@
  * Once submitted, a TokenCustomerID is provided which can be
  * used in future transactions instead of the card details.
```

Comment:
Can I actually get you to remove all the vendor updates & just do the composer.json patch instead & I'll run composer.json & push that. If you have had to patch the vendor extension then it's ok to link your own fork rather than the upstream of the extension - but at least that way the patch history on your fork will be visible. As it is I can't quite tell what is core code upstream & what you have changed